src/components/Menu/index.jsx

```diff
@@ -0,0 +1,60 @@
+import { useDispatch } from 'react-redux';
+import { CloseIcon, MenuIcon } from '../Image/styles';
+import {
+  CloseButton,
+  MenuButton,
+  MenuButtonsContainer,
+  MenuHeader,
+  MenuView,
+} from './style';
+import { closeMenu } from '../../store/menu/menu';
+import { useNavigate } from 'react-router-dom';
+import { BarContainer, KlarnaName } from '../Navbar/style';
+
+function Menu() {
+  const dispatch = useDispatch();
+  const navitage = useNavigate();
+
+  const handleClose = () => {
+    dispatch(closeMenu());
+  };
+
+  const handleHome = () => {
```

Comment:
This and the next 3 functions can be all one function with url parameter.
There is a typo - write navigate